2025-08-13 14:56:06.453 [ERROR] Error: connect ECONNREFUSED ::1:3306
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
    --------------------
    at Protocol._enqueue (C:\Users\霽秋\OneDrive\桌面\demo\testAPI-master\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (C:\Users\霽秋\OneDrive\桌面\demo\testAPI-master\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at PoolConnection.connect (C:\Users\霽秋\OneDrive\桌面\demo\testAPI-master\node_modules\mysql\lib\Connection.js:116:18)
    at Pool.getConnection (C:\Users\霽秋\OneDrive\桌面\demo\testAPI-master\node_modules\mysql\lib\Pool.js:48:16)
    at C:\Users\霽秋\OneDrive\桌面\demo\testAPI-master\node_modules\typeorm\driver\mysql\MysqlDriver.js:1011:18
    at new Promise (<anonymous>)
    at MysqlDriver.createPool (C:\Users\霽秋\OneDrive\桌面\demo\testAPI-master\node_modules\typeorm\driver\mysql\MysqlDriver.js:1008:16)
    at MysqlDriver.connect (C:\Users\霽秋\OneDrive\桌面\demo\testAPI-master\node_modules\typeorm\driver\mysql\MysqlDriver.js:305:36)
    at DataSource.initialize (C:\Users\霽秋\OneDrive\桌面\demo\testAPI-master\node_modules\typeorm\data-source\DataSource.js:137:27)
    at DataSource.connect (C:\Users\霽秋\OneDrive\桌面\demo\testAPI-master\node_modules\typeorm\data-source\DataSource.js:176:21) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '::1',
  port: 3306,
  fatal: true
}
2025-08-13 14:56:59.887 [ERROR] Error: connect ECONNREFUSED ::1:3306
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
    --------------------
    at Protocol._enqueue (C:\Users\霽秋\OneDrive\桌面\demo\testAPI-master\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (C:\Users\霽秋\OneDrive\桌面\demo\testAPI-master\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at PoolConnection.connect (C:\Users\霽秋\OneDrive\桌面\demo\testAPI-master\node_modules\mysql\lib\Connection.js:116:18)
    at Pool.getConnection (C:\Users\霽秋\OneDrive\桌面\demo\testAPI-master\node_modules\mysql\lib\Pool.js:48:16)
    at C:\Users\霽秋\OneDrive\桌面\demo\testAPI-master\node_modules\typeorm\driver\mysql\MysqlDriver.js:1011:18
    at new Promise (<anonymous>)
    at MysqlDriver.createPool (C:\Users\霽秋\OneDrive\桌面\demo\testAPI-master\node_modules\typeorm\driver\mysql\MysqlDriver.js:1008:16)
    at MysqlDriver.connect (C:\Users\霽秋\OneDrive\桌面\demo\testAPI-master\node_modules\typeorm\driver\mysql\MysqlDriver.js:305:36)
    at DataSource.initialize (C:\Users\霽秋\OneDrive\桌面\demo\testAPI-master\node_modules\typeorm\data-source\DataSource.js:137:27)
    at DataSource.connect (C:\Users\霽秋\OneDrive\桌面\demo\testAPI-master\node_modules\typeorm\data-source\DataSource.js:176:21) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '::1',
  port: 3306,
  fatal: true
}
